lib/common/patchset.c

```diff
@@ -768,7 +768,7 @@ process_v1_additions(xmlNode *parent, xmlNode *target, xmlNode *patch)
  * \return TRUE if format is valid, FALSE if invalid
  */
 static bool
-find_patch_xml_node(xmlNode *patchset, int format, bool added,
+find_patch_xml_node(const xmlNode *patchset, int format, bool added,
```

Comment:
Bump the copyright

Reply:
Done, also rebased on current main. No other changes.